app/components/learnergroup-members.js

```diff
@@ -1,7 +1,19 @@
 import Ember from 'ember';
 import DS from 'ember-data';
+import InPlace from 'ilios/mixins/inplace';
+
+const { computed, isEmpty, ObjectProxy, observer, RSVP } = Ember;
+const { all } = RSVP;
+
+export default Ember.Component.extend(InPlace, {
```

Comment:
OK, I see why you're using InPlace here, but I don't think this functionality is going to get used anywhere else so it should all just be placed in the component.  Am I wrong about that?